docs/docs/topics/transformation.mdx

```diff
@@ -94,6 +94,12 @@ A TransformPython can be rendered with 2 different methods:
 - As part of an [Artifact](./artifact.mdx)
 - Via the CLI for development and troubleshooting: [infrahubctl transform]($(base_url)infrahubctl/infrahubctl-transform)
 
+## Targeting transformations with groups
```

Comment:
I find this section a bit confusing because technically Transform are not specific to a group
An artifact definition is bringing together a transform and a group

Reply:
Reworked it, let me know if this works.